sentry-opentelemetry/sentry-opentelemetry-extra/src/main/java/io/sentry/opentelemetry/OtelSpanWrapper.java

```diff
@@ -217,6 +217,7 @@ private void updateBaggageValues() {
             });
         baggage.setValuesFromTransaction(
             getSpanContext().getTraceId(),
+            null,
```

Comment:
should we also read out the replayId from scope here and pass it instead of `null`?

Reply:
We can.